modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -87,6 +87,10 @@ def add_to_apply_calls(self, func, *args, **kwargs):
             Function to be added to the call queue.
         *args : iterable
             Additional positional arguments to be passed in `func`.
+        length : reference or int, optional
+            Length or reference to it of wrapped ``pandas.DataFrame``.
```

Comment:
```suggestion
            Length, or reference to length, of wrapped ``pandas.DataFrame``
```

I think you're copying this from the other `add_to_apply_calls` docstrings, but I find the current wording very unclear.